docs/DeveloperGuide.md

```diff
@@ -383,6 +383,45 @@ The sequence diagram is as follows:
 
 The exported file can be subsequently imported back into any other instance of ContaX, similar to the existing `.json` system of import/export.
 
+### Enhanced Find Logic
+
+The previous implementation find target is limited to name field only. Hence, extending the find function will increase the user experience by enable them easily search `Person` with `phone`
+`email` and `address` field.
+
+This is done by creating `searchType` model within the `model util` package and extracting the original `NameConstrainsKeywordsPreicate` to form `ConstrainsKeywordsPreicate` then create different inherit of three different types of predicate.
```

Comment:
Please follow java standards when it comes to classes and packages. Can you also check your class names, as per  the previous comment by @Wei-Xinn. The last part of the sentence also completely does not make sense.
```suggestion
This is done by creating a helper `SearchType` model within the `model.util` package and abstracting the original `NameConstrainsKeywordsPreicate` to form `ConstrainsKeywordsPreicate` then create different inherit of three different types of predicate.
```